.github/workflows/android_deploy_production.yml

```diff
@@ -0,0 +1,57 @@
+name: Android - Deploy Production build to Firebase
```

Comment:
Please remove this workflow after testing 🙏

Reply:
I think we should keep it. The idea is to have a mirror workflow to verify the CD workflow in the generated project. I tried to find a way to trigger the workflow inside the `template` project but no luck yet because we need to execute the project generating step first. Hence, the mirror workflow has duplicated steps with the original workflow in the `template` project.

What do you think? @doandinh @manh-t @chornerman